### DISCUSSION TITLE
For the occasional network situation, Could you consider to offer a version without pictures?

### DISCUSSION THREAD

**Discussion by UncleK** [willwulfken/MidJourney-Styles-and-Keywords-Reference#46] [General]
For the occasional network situation, Could you consider to offer a version without pictures?
Just the keywords of the styles would be okay, such as an excel sheet.
That would be helpful to get a quick reference.
Thaks a lot!

**Comment by willwulfken** [comment]
> For the occasional network situation, Could you consider to offer a version without pictures? Just the keywords of the styles would be okay, such as an excel sheet. That would be helpful to get a quick reference. Thaks a lot!

It took me a while to get around to it (I've been very busy as of late) but I finally did: https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/releases/tag/1.0
